contracts/linear/src/events.rs

```diff
@@ -118,9 +118,9 @@ pub enum Event<'a> {
         weight: u16,
     },
     ValidatorUpdatedWeight {
```

Comment:
```suggestion
    ValidatorsUpdatedWeights {
```

maybe rename the event?